src/rsz/test/repair_wire10.ok

```diff
@@ -9,6 +9,7 @@
 [INFO ODB-0132]     Created 2 special nets and 6 connections.
 [INFO ODB-0133]     Created 4 nets and 6 connections.
 [INFO IFP-0001] Added 142 rows of 11052 site FreePDK45_38x28_10R_NP_162NW_34O with height 1.
+[WARNING DPL-0038] No 1-site fill cells detected.  To remove 1-site gaps use the -disallow_one_site_gaps flag.
```

Comment:
I don't see why we get this message for ng45.  FILLCELL_X1 is a one site filler afaik.

Reply:
That's a really good catch. I ended up discovering a bug that affected 10s of tests for this warning.

That was the logic of the checker, and FILLCELL_X1 is indeed a core spacer and of width equal to that of one site.

```Cpp
// Return true if cell is a single site Core Spacer.
bool Opendp::isOneSiteCell(odb::dbMaster* db_master) const
{
  return db_master->getType() == odb::dbMasterType::CORE_SPACER
         && db_master->getWidth() == site_width_;
}
```

The bug was that the site_width_ was 0.  I assumed it gets in the constructor or something in the initialisation, but it  was set in the  examineRows() method.
Because I was unaware of this, simply calling  checkOneSiteDbMaster() before examineRows() resulted in a lot of warnings. 

reordering them fixes the issue, but it makes me think if having site_width_ being set in the examineRows was a good idea or not.

Also, would it ever be the case that we have different site_width for different rows? for example as we did in multi-height rows but for width instead of height? If that might ever be the case, then the last row that examineRows iterate over would be the site_width_ used to check all one_site_cells

Reply:
In theory you could have different rows with different sites and they could have different widths (likely in different regions).  In practice we don't support that so I don't think we need to worry about it (many other things would also break too).